stacks-signer/src/signerdb.rs

```diff
@@ -109,6 +132,13 @@ CREATE TABLE IF NOT EXISTS signer_states (
     encrypted_state BLOB NOT NULL
 )";
 
+const CREATE_BURN_STATE_TABLE: &str = "
+CREATE TABLE IF NOT EXISTS burn_blocks (
+    block_hash TEXT PRIMARY KEY,
+    block_height INTEGER,
```

Comment:
Should these be `NOT NULL`?  Also, should we be using `STRICT` by default?

Reply:
Sure, added in [eefedbb](https://github.com/stacks-network/stacks-core/pull/4956/commits/eefedbb21e969f8f84f1d0e6a56c3b73b2b10504)